README.md

```diff
@@ -8,6 +8,32 @@ pyXDF
 
 pyXDF is a Python importer for [XDF](https://github.com/sccn/xdf) files.
 
+## Sample usage
+
+``` python
+import pyxdf
+import matplotlib.pyplot as plt
+import numpy as np
+
+data, header = pyxdf.load_xdf('test.xdf')
+
+for stream in data:
+    y = stream['time_series']
+
+    if isinstance(y, list):
+        # list of strings, draw one vertical line for each marker
+        for timestamp, marker in zip(stream['time_stamps'], y):
+            plt.axvline(x=timestamp)
+            print(f'Marker "{marker}" @ {timestamp}s')
```

Comment:
Maybe use `print(f'Marker {marker[0]} @ {timestamp:.2f}s')` for a nicer display?

Reply:
Or if you want the quotes around the marker name:

```python
print(f'Marker "{marker[0]}" @ {timestamp:.2f}s')
```